### PR TITLE
Ci win1 and checks

### PR DESCRIPTION
**Pull request by rasapala**
### 🛠 Summary

JIRA/Issue if applicable.
Describe the changes.

### 🧪 Checklist

- [ ] Unit tests added.
- [ ] The documentation updated.
- [ ] Change follows security best practices.
``

### CODE REVIEW

File: docs/windows_binary_guide.md [hunk 1]
```diff
@@ -1,16 +1,32 @@
...
-## No REST API support
+# OpenVINO&trade; Model Server Deployment Guide for Windows 11 (Experimental/Alpha)
+This document describes experimental/alpha windows deployment for ovms.exe binary.
+Follow the instructions once you have compiled ovms.exe and you want to deploy the binary on a Windows 11 system.
+
+OpenVINO&trade; Model Server is in experimental/alpha stage of windows enabling with limited functionality and quality.
+It is recommended to use the top of main repository branch for more feature enabled code and better software quality for windows.
+
+## List of enabled features:
+### Limited model server basic functionality
```
Comment (dtrawins) [inline docs/windows_binary_guide.md:9]: are there some limitations beside disabled features?

File: docs/windows_binary_guide.md [hunk 2]
```diff
@@ -1,16 +1,32 @@
...
+### GRPC API
```
Comment (dtrawins) [inline docs/windows_binary_guide.md:10]: serving single models in all formats
DAG pipelines

File: docs/windows_binary_guide.md [hunk 3]
```diff
@@ -1,16 +1,32 @@
...
+### Mediapipe graphs execution
+
+## List of disabled features:
+### Full ovms feature parity with Linux implementation
```
Comment (dtrawins) [inline docs/windows_binary_guide.md]: ? It is not a parity.

File: docs/windows_binary_guide.md [hunk 4]
```diff
@@ -1,16 +1,32 @@
...
+### No LLM support
```
Comment (dtrawins) [inline docs/windows_binary_guide.md]: I would drop No phase. It is a list of limitations anyway.

**Review by dtrawins — APPROVED** [review]

**Review by atobiszei — APPROVED** [review]

**Pull request merged**